internal/server/routes.go

```diff
@@ -126,7 +127,11 @@ func (h scrapeServer) mustHomeTemplate() *template.Template {
 			return s
 		}
 	}
-	tmpl = tmpl.Funcs(template.FuncMap{"AuthToken": keyF})
+	funcMap := template.FuncMap{
+		"AuthToken":   authTokenF,
+		"AuthEnabled": authEnabledF,
```

Comment:
This adds the hook so we can skip the token entry widget when auth is not enabled